scss/_root.scss

```diff
@@ -142,8 +142,8 @@
 // scss-docs-start root-dark-rule
 // Boosted mod
 [class*="bg-black"],
-[class*="-dark"]:not(.border-dark):not(.text-dark):not(.btn-dark):not(.focus-ring-dark):not(.link-underline-dark):not(.link-dark),
-[class*="bg-secondary"] {
+[class*="-dark"]:not(.border-dark):not(.text-dark):not(.btn-dark):not(.focus-ring-dark):not(.link-underline-dark):not(.link-dark):not(.text-bg-dark),
+[class*="bg-secondary"]:not(.text-bg-secondary) {
```

Comment:
TBH, I'd rather change the card Sass variables instead of adding some rules here since here it could have much impacts rather than just cards.

Reply:
Yep, we consider right now (won't be the case when the dark mode = dark variant) that `.bg-secondary`, `.bg-black` and `.bg-dark` contain dark variants. `.text-bg-dark` is a derivative of `.bg-dark` so might be considered as something containing dark variants as well.
Let's revisit with the actual dark mode in Boosted because `.bg-dark|secondary|black` will have their real role: just a colored background that doesn't change whatever the color mode.

In the meantime, I'd say it forces us to do the following (which won't work perfectly for the dark mode and that's not an issue right now):
```scss
$card-cap-bg:                       $black !default; // Boosted mod: instead of `rgba(var(--#{$prefix}body-color-rgb), .03)`
$card-cap-color:                    $white !default; // Boosted mod: instead of `null`
```